api/location/models.py

```diff
@@ -11,15 +11,18 @@ class CountryType(enum.Enum):
     Uganda = "Uganda"
     Kenya = "Kenya"
     Nigeria = "Nigeria"
+    Kigali = "Kigali"
```

Comment:
I think the country type here is `Rwanda`. You can consider checking into that.

Reply:
The story assigned reads Kigali, but I will look into it, thank you.